database/mint.go

```diff
@@ -49,3 +49,20 @@ WHERE mint_params.height <= excluded.height`
 
 	return nil
 }
+
+// GetMintParams allows to get the current mint params
+func (db *Db) GetMintParams() (creminttypes.Params, error) {
```

Comment:
This function throws a panic when there is no mint params stored inside the db, can you add the handler for it pls?